packages/trader/src/Modules/Trading/Components/Form/screen-large.tsx

```diff
@@ -22,16 +25,11 @@ const ScreenLarge = ({ is_market_closed, is_trade_enabled }) => (
                 </Fieldset>
                 <TradeParams />
                 <div className='purchase-container'>
-                    <Purchase is_market_closed={is_market_closed} />
+                    <Purchase is_market_closed={!!is_market_closed} />
```

Comment:
@kate-deriv Instead you can give it default value of false, Whenever you have a optional boolean prop give it a default value of false 🙇🏻

Reply:
Thank you, will apply